src/components/Tables/Table.tsx

```diff
@@ -86,41 +109,83 @@ const Table: React.FC<Props> = (props) => {
         ([headerKey, headerName]) => {
             return {
                 name: headerName,
                 selector: (row: Row) => row[headerKey],
                 sortable: true,
+                cell(row, rowIndex, column, id) {
+                    const tooltip = data[rowIndex].tooltips?.[headerKey];
+                    const tooltipElement = tooltip ? (
+                        <>
+                            <Spacer />
+                            <SpeechBubbleIcon onHoverText={tooltip} />
+                        </>
+                    ) : null;
+                    return (
+                        <RowDiv key={id}>
+                            {row[headerKey]}
+                            {tooltipElement}
+                        </RowDiv>
+                    );
+                },
             };
         }
     );
 
-    columns.unshift({
-        name: (
-            <input
-                type="checkbox"
-                aria-label="Select all rows"
-                checked={selectAllCheckBox}
-                onClick={toggleAllCheckBox}
-            />
-        ),
-        cell: (row: Row) => (
-            <input
-                type="checkbox"
-                aria-label={`Select row ${row.rowId}`}
-                checked={selectCheckBoxes[row.rowId]}
-                onClick={() => toggleOwnCheckBox(row.rowId)}
-            />
-        ),
-        width: "47px",
-    });
+    if (props.checkboxes) {
+        columns.unshift({
+            name: (
+                <input
+                    type="checkbox"
+                    aria-label="Select all rows"
+                    checked={selectAllCheckBox}
+                    onClick={toggleAllCheckBox}
+                />
+            ),
+            cell: (row: Row) => (
+                <input
+                    type="checkbox"
+                    aria-label={`Select row ${row.rowId}`}
+                    checked={selectCheckBoxes[row.rowId]}
+                    onClick={() => toggleOwnCheckBox(row.rowId)}
+                />
+            ),
+            width: "47px",
+        });
+    }
+
+    if (props.reorderable) {
+        columns.unshift({
+            name: <></>,
+            cell: (row: Row, rowIndex, column, id) => (
+                <ReorderArrowDiv>
+                    <StyledIcon onClick={() => {
+                        if (row.rowId === 0) return;
+                        setData(data => {
+                            const newData = [...data];
+                            const temp = newData[row.rowId];
+                            newData[row.rowId] = newData[row.rowId - 1];
+                            newData[row.rowId - 1] = temp;
+                            return newData;
+                        });
+                    }} icon={faAnglesUp} />
+                    <StyledIcon onClick={() => {
+                        if (row.rowId === data.length - 1) return;
+                        setData(data => {
```

Comment:
potential to export this setData to a new function?

Reply:
Done